src/lib/Scenes/City/City.tsx

```diff
@@ -1,52 +1,59 @@
 import { Box, color, Flex, Serif } from "@artsy/palette"
+import { GlobalMap_viewer } from "__generated__/GlobalMap_viewer.graphql"
 import React, { Component } from "react"
+import { ScrollView } from "react-native"
 import styled from "styled-components/native"
 import { EventEmitter } from "../Map/EventEmitter"
-
-export interface City {
-  name: string
-  epicenter: {
-    lat: number
-    lng: number
-  }
-}
+import { Tab } from "../Map/types"
+import { AllEvents } from "./Components/AllEvents"
 
 interface State {
-  shows: any[]
-  fairs: any[]
-  title: string
+  city?: GlobalMap_viewer
+  filter: Tab
 }
 
-export class CityView extends Component<null, State> {
+export class CityView extends Component<{}, State> {
   state = {
-    shows: [],
-    fairs: [],
-    title: "",
+    city: null,
+    filter: { id: "all", text: "All events" },
   }
 
   componentWillMount() {
     EventEmitter.subscribe("map:change", ({ filter, city }) => {
-      const { shows, fairs } = city
       console.log(city)
```

Comment:
I'm assuming this is still needed?

Reply:
It's not _needed_ but it does help give us an idea of the shape of the data while we build the UI.